tools/constants.mjs

```diff
@@ -26,10 +26,10 @@ export const AO_LOADER_HANDLER_ENV = {
 };
 
 export const AO_LOADER_OPTIONS = {
-  format: 'wasm64-unknown-emscripten-draft_2024_02_15',
+  format: 'wasm32-unknown-emscripten-metering',
   inputEncoding: 'JSON-1',
   outputEncoding: 'JSON-1',
-  memoryLimit: '524288000', // in bytes
+  memoryLimit: '1073741824', // in bytes
```

Comment:
Better comment might be `1 GiB`

Reply:
[091b4fc](https://github.com/ar-io/ar-io-ant-process/pull/54/commits/091b4fcb8de8f2f1a342d2578d225f6af707f2c6)